R/class_matlist.R

```diff
@@ -47,14 +51,14 @@ valid.matlist <- function(object) {
   }
   
   if(!x4) {
-    y <- which(!sapply(object@data, det) > 0)
+    y <- which(!vapply(object@data, det, 1.23) > 0)
```

Comment:
Why 1.23?

Reply:
Just to indicate that we're expecting a single numeric value.  I updated the call to name the arguments.